### PR TITLE
Ubuntu Server install: revised words and images.

### LINKED ISSUE #253: Review - install ubuntu server

### PR DESCRIPTION
**Pull request by degville**
fixes #253

### CODE REVIEW

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 1]
```diff
@@ -1,64 +1,164 @@
...
+
+![screenshot](https://assets.ubuntu.com/v1/bde68a78-server-partition-auto.png)
+
+positive
+: If your server storage is already being used, or has been used prior to the installation, you will get a slightly different menu. Once again, the simplest option is to select 'Guided - use the entire disk and set up LVM', but you can also choose the specific partition if that's convenient.
+
+## Package retrieval
+Duration: 0:02
+
+After accepting the changes that are going to be made to your storage, Ubuntu Server will be assemble the packages to be installed. This will take a few moments.
```
Comment (pmatulis) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: the installer will determine the packages to be installed?

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 2]
```diff
@@ -1,64 +1,164 @@
...
-# Installing Ubuntu Server for general use
+# Install Ubuntu Server
 
-## Getting started
-Duration: 0:05
+## Overview
+Duration: 0:01
 
-The basic steps to install Ubuntu Server from CD or USB stick are the same for installing any operating system. Unlike the desktop version, Ubuntu Server does not include a graphical installation program. Instead, it uses a console menu-based process.
+Ubuntu server is Ubuntu tailored for networks and services, and it's just as capable sharing files on your home network as it is running a 50,000 render farm.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: I took this example from https://www.ubuntu.com/server to present a unified message. But I agree, it's a little obscure. I'll change it.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 3]
```diff
@@ -1,64 +1,164 @@
...
 
-positive
-: Note - this is a very high-level guide. For a more detailed guide go to the [Ubuntu Server help documentation](https://help.ubuntu.com/lts/serverguide/installation.html
+Unlike the desktop version of Ubuntu, Ubuntu Server does not include a graphical installation program. Instead, it uses a console menu-based process. If you'd rather install the desktop version, take a look at our [Install Ubuntu desktop](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-deskto) tutorial.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: ok

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 4]
```diff
@@ -1,64 +1,164 @@
...
+
+This guide will be a brief overview of the installation from either a DVD or a USB flash drive, suitable for anyone with an interest in running their own server. For a more detailed guide on Ubuntu Server's capabilities and its configuration, take a look at our [Ubuntu Server help documentation](https://help.ubuntu.com/lts/serverguide/installation.html
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: thanks!

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 5]
```diff
@@ -1,64 +1,164 @@
...
-## Prepare a DVD or USB flash drive
-Duration: 0:10
+![screenshot](https://assets.ubuntu.com/v1/fcc62c28-server-welcome.png)
+
+## Requirements
+Duration: 0:01
+
+You'll need to consider the following before starting the installation:
+
+* Ensure you have at least 2GB of free storage space.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: I took those details from the online docs. Also, for format followed the same as the desktop install for consistency. The idea is to keep everything to a minimum (if you see how these tutorials are presented on t.u.c. - take a look at the original.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 6]
```diff
@@ -1,64 +1,164 @@
...
-Duration: 0:10
+![screenshot](https://assets.ubuntu.com/v1/fcc62c28-server-welcome.png)
+
+## Requirements
+Duration: 0:01
+
+You'll need to consider the following before starting the installation:
+
+* Ensure you have at least 2GB of free storage space. 
+* Have access to either a DVD or a USB flash drive containing the version of Ubuntu Server you want
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: That's a separate tutorial.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 7]
```diff
@@ -1,64 +1,164 @@
...
+  to install.
+* Make sure you have a recent backup of your data. While it's unlikely that
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: this is all for folks new to installing Ubuntu server. It's kind of a caveat. I'll add some clarification.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 8]
```diff
@@ -1,64 +1,164 @@
...
+
+See [Installation/System Requirements](https://help.ubuntu.com/community/Installation/SystemRequirements#Ubuntu_Server_.28CLI.29_Installation) for more specific details on hardware requirements. We also have [several tutorials](/) that explain how to create an Ubuntu DVD or USB flash drive.
+
+
+## Boot from DVD
+Duration: 0:02
+
+To trigger the installation process, perform the following:
+
+1. Put the Ubuntu DVD into your optical/DVD drive.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: what if it's a Blu-ray drive?

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 9]
```diff
@@ -1,64 +1,164 @@
...
+1. Restart your computer.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: I think that's too technical for this level.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 10]
```diff
@@ -1,64 +1,164 @@
...
+
+![screenshot](https://assets.ubuntu.com/v1/87351452-server-language-select.png)
 
-First, download and burn the ISO file from the [Ubuntu Server download page](https://www.ubuntu.com/download/server) and, if you want, [verify the download](/tutorial/tutorial-how-to-verify-ubuntu).
+If you don’t get this menu, read the [booting from the DVD guide](https://help.ubuntu.com/community/BootFromCD?_ga=2.102380610.2115462233.1496186978-1155966827.1485186360) for more information.
 
+## Boot from USB flash drive
+Duration: 0:02
 
+Most computers will boot from USB automatically. Simply insert the USB flash drive and either power on your computer or restart it. You should see the same language menu we saw in the previous 'Install from DVD' step, followed by the boot menu.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: I'm not sure we should bring the BIOS into this.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 11]
```diff
@@ -1,64 +1,164 @@
...
+Duration: 0:20
+
+The Ubuntu Server boot menu includes handy options for testing your system and for testing the validity of the install media and system disks.
+
+There are two options for installation, and you should select 'Install Ubuntu Server'. 
+
+![screenshot](https://assets.ubuntu.com/v1/c08110df-server-boot-menu.png)
+
+positive
+: The second boot option, 'Install Ubuntu Server with the HWE kernel', replaces the kernel at the core of the operating system with a kernel that gets more frequent updates. This can be useful if you are running cutting edge or new hardware, at the expense of potential stability.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: thanks. I think it's best if I remove any mention of HWE.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md [hunk 12]
```diff
@@ -1,64 +1,164 @@
...
 
-There are many ways disk layout can be configured. For detailed information please read the [Installing from CD](https://help.ubuntu.com/lts/serverguide/installing-from-cd.html) documentation.
+## User configuration
+Duration: 0:03
 
-## Done.
+After networking, you'll be asked to enter your full name, username and password. As you're configuring a server that's likely to be accessible from the internet (if you choose), make sure your password is strong and difficult to guess.
 
-The Ubuntu base system is then installed.
+Similarly, you get the option to encrypt your home directory and we'd recommend saying yes to this. However, this will mean your personal data will be irretrievable if you lose or forget your password.
```
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: Excellent point, thanks.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md (hunk 1 above)
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: I'll cut this fluff, thanks.

File: tutorials/install-ubuntu-server/install-ubuntu-server.md (hunk 8 above)
Comment (degville) [inline tutorials/install-ubuntu-server/install-ubuntu-server.md]: I'll remove this :)

**Comment by pmatulis** [comment]
Thank you! :ice_cream:

**Pull request merged**